app-main/public/css/style.css

```diff
@@ -533,11 +534,88 @@ form {
 
 @media only screen and (max-width: 1090px) {
   body {
-    min-width: 1090px;
+    /* min-width: 1090px; */
```

Comment:
Can you remove the commented out code please? (Same for rest of file)

Reply:
Yh, I'll remove them.